--- bzr-2.7.0/bzrlib/transport/http/_urllib2_wrappers.py.orig	2016-03-18 19:08:49.765967428 +0100
+++ bzr-2.7.0/bzrlib/transport/http/_urllib2_wrappers.py	2016-03-18 19:35:19.875900698 +0100
@@ -88,6 +88,7 @@
 # Note for packagers: if there is no package providing certs for your platform,
 # the curl project produces http://curl.haxx.se/ca/cacert.pem weekly.
 _ssl_ca_certs_known_locations = [
+    u'/etc/certs/ca-certificates.crt', # PLD Linux
     u'/etc/ssl/certs/ca-certificates.crt',  # Ubuntu/debian/gentoo
     u'/etc/pki/tls/certs/ca-bundle.crt',  # Fedora/CentOS/RH
     u'/etc/ssl/ca-bundle.pem',  # OpenSuse
